ipc/hypervisor/src/service.rs.in

```diff
@@ -42,7 +42,7 @@ pub struct ModuleState {
 
 #[derive(Ipc)]
 pub trait ControlService {
-	fn shutdown(&self);
+	fn shutdown(&self) -> bool;
```

Comment:
Having a return type forces invocation to be synchronous
Until the exact issue will be found